apps/web/app/availability/[schedule]/page.tsx

```diff
@@ -46,47 +47,47 @@ const Page = async ({ params, searchParams }: PageProps) => {
   if (!parsed.success) {
     notFound();
   }
-  // const scheduleId = Number(params.schedule);
-
-  // const session = await getServerSessionForAppDir();
-  // const userId = session?.user?.id;
-  // if (!userId) {
-  //   notFound();
-  // }
-
-  // let userData, schedule, travelSchedules;
-
-  // try {
-  //   userData = await UserRepository.getTimeZoneAndDefaultScheduleId({
-  //     userId,
-  //   });
-  //   if (!userData?.timeZone || !userData?.defaultScheduleId) {
-  //     throw new Error("timeZone and defaultScheduleId not found");
-  //   }
-  // } catch (e) {
-  //   notFound();
-  // }
-
-  // try {
-  //   schedule = await ScheduleRepository.findDetailedScheduleById({
-  //     scheduleId,
-  //     isManagedEventType: false,
-  //     userId,
-  //     timeZone: userData.timeZone,
-  //     defaultScheduleId: userData.defaultScheduleId,
-  //   });
-  // } catch (e) {}
-
-  // try {
-  //   travelSchedules = await TravelScheduleRepository.findTravelSchedulesByUserId(userId);
-  // } catch (e) {}
+  const scheduleId = parsed.data.schedule;
 
-  return (
-    <AvailabilitySettingsWebWrapper
+  const session = await getServerSessionForAppDir();
+  const userId = session?.user?.id;
+  if (!userId) {
+    notFound();
+  }
 
-    // scheduleFetched={schedule} travelSchedules={travelSchedules}
-    />
-  );
+  try {
+    const userData = await UserRepository.getTimeZoneAndDefaultScheduleId({
+      userId,
+    });
+    if (!userData?.timeZone || !userData?.defaultScheduleId) {
+      throw new Error("timeZone and defaultScheduleId not found");
+    }
+
+    const schedule = await ScheduleRepository.findDetailedScheduleById({
+      scheduleId,
+      isManagedEventType: false,
+      userId,
+      timeZone: userData.timeZone,
+      defaultScheduleId: userData.defaultScheduleId,
+    });
+    const revalidatePage = async () => {
+      "use server";
+      revalidatePath(`availability/${scheduleId}`);
+    };
+    // try {
+    //   travelSchedules = await TravelScheduleRepository.findTravelSchedulesByUserId(userId);
+    // } catch (e) {}
```

Comment:
- this replaces `trpc.viewer.getTravelSchedules.useQuery` in the schedule-view client component.
- let's uncomment this in a separate PR because this isn't related to the flickering issue